views/admin.php.inc

```diff
@@ -1,183 +1,173 @@
+                                <input type="hidden" class="hidden" name="widget_id" value="<?php echo $widget['widget_id']?>">
                             </div>
-                        </div>
+                        </fieldset>
+                        <form class="tawkto-widget-settings" action="" method="post" id="module_form" accept-charset="UTF-8">
+                            <fieldset class="form-wrapper" id="visibility-settings">
+                                <legend><span class="fieldset-legend">Visibility Settings</span></legend>
+                                <div class="fieldset-wrapper">
+                                    <div class="fieldset-description">Define where the Tawk.to widget will and won't show</div>
+                                    <div class="form-item form-type-checkbox">
+                                        <?php
+                                            $checked = true;
+                                            if (!is_null($display_opts)) {
+                                                if (!$display_opts->always_display) {
+                                                    $checked = false;
+                                                }
+                                            }
+                                        ?>
+                                        <input type="checkbox" id="always_display" name="always_display" value="1" class="checkbox" 
+                                            <?php echo ($checked)?'checked':'';?> />
+                                        <label class="option" for="always_display">Always show Tawk.To widget on every page</label>
+										<div class="description">
+                                            Select to show on all except the exceptions<br>
+											De-select to select the specific pages
+                                        </div>
+                                    </div>
+                                    <div class="form-item form-type-textarea div_hide_specific">
+                                        <label for="hide_oncustom">Except on pages:</label>
+                                        <div class="form-textarea-wrapper resizable textarea-processed resizable-textarea">
+                                            <textarea class="form-textarea hide_specific" name="hide_oncustom" id="hide_oncustom" cols="40" rows="6"><?php if (!empty($display_opts->hide_oncustom)) {
+                                                    $whitelist = json_decode($display_opts->hide_oncustom);                                                     
+                                                    foreach ($whitelist as $page) { echo $page."\r\n"; }
+                                            } else {
+                                            } ?></textarea>
+                                            <div class="grippie"></div>
```

Comment:
The grippie doesn't work so it can be removed if not needed.

Reply:
Removed.  I saw it work on another admin page and thought it was a good idea but the bit of JS powering it does not seem to be universal